.brazil.json

```diff
@@ -102,7 +102,8 @@
         "test-utils": { "skipImport": true },
         "tests-coverage-reporting": { "skipImport": true },
         "third-party": { "skipImport": true },
-        "third-party-slf4j-api": { "skipImport": true }
+        "third-party-slf4j-api": { "skipImport": true },
+        "migration-tool-tests": {"migration-tool-tests":  true}
```

Comment:
is this supposed to be `"skipImport": true`?

Reply:
Good catch!